topcoffea/modules/eft_helper.py

```diff
@@ -7,35 +7,33 @@
 import math
 
 @numba.njit
-def calc_eft_weights(q_coeffs,wc_values):
+def calc_eft_weights(q_coeffs, wc_values):
     """Calculate the weights for a specific set of WC values.
 
     Args:
         q_coeffs: Array specifying a set of quadric coefficients parameterizing the weights.
-                  The last dimension should specify the coefficients, while any earlier dimensions
-                  might be for different histogram bins, events, etc.
+                The last dimension should specify the coefficients, while any earlier dimensions
```

Comment:
Could you revert all of the formatting updates, so that the only changes are to content?

Reply:
Done